### PR TITLE
draft of geckoman

### PR DESCRIPTION
**Pull request by chrisatconcur**

### CODE REVIEW

File: superhero.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Superhero Design Doc
+
+## Stats
+Name: Steve
+Sex: Male  
+Country of Origin: USA  
+
+## History
+Bob was an ordinary fellow until he got bit by a magical gecko. At which point
```
Comment (chrisleedotninja) [inline superhero.md]: I think the Magical superpowers thing is over done.. maybe the gecko could be science based or alien? Also... Geckos are cool but not badass.. Maybe we could do something else... like a spider?

File: superhero.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+he got gecko like super powers!
+
+## Powers
+### Gecko Sticky
```
Comment (chrisleedotninja) [inline superhero.md]: If we change it from a gecko to a spider... we should probably change the powers around.

File: superhero.md (hunk 1 above)
Comment (chrisatconcur) [reply superhero.md]: I'd be down for changing it from a gecko to a spider... maybe a radioactive spider?

File: superhero.md (hunk 2 above)
Comment (chrisatconcur) [reply superhero.md]: I'll think through some new powers...

File: superhero.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ Sex: Male
 Country of Origin: USA  
 
 ## History
-Bob was an ordinary fellow until he got bit by a magical gecko. At which point
-he got gecko like super powers!
+Bob was an ordinary fellow until he got bit by a radioactive spider. At which point
+he got spider like super powers!
 
 ## Powers
```
Comment (chrisleedotninja) [inline superhero.md]: These Spider powers look awesome! I like it.

File: superhero.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Superhero Design Doc
+
+## Stats
+Name: Steve
```
Comment (chrisleedotninja) [inline superhero.md]: Hey... maybe we shouldn't name this guy Steve... What about peter? Peter Parker? And what's his superhero name going to be?

Comment (chrisatconcur) [reply superhero.md]: Yeah... Peter Parker sounds good... how about 'Spiderman' for a super hero name?

Comment (chrisleedotninja) [reply superhero.md]: Yeah.. Let's go with Spider Man!

**Pull request merged**